sycl/include/sycl/queue.hpp

```diff
@@ -1059,6 +1059,124 @@ class __SYCL_EXPORT queue {
         CodeLoc);
   }
 
+  /// Copies data from a memory region pointed to by a placeholder accessor to
+  /// another memory region pointed to by a shared_ptr.
+  ///
+  /// \param Src is a placeholder accessor to the source memory.
```

Comment:
Not sure the API works with placeholder accessors only, does it?

Reply:
Technically it should be as this is a function on queue, so at this level it should only be possible to have placeholder and host accessors, the latter being moved to `host_accessor`. We could have a check to make sure it is not a host accessor in the meantime.